### DISCUSSION TITLE
rewiring the CR-10s PRO screen to work with a SKR 2

### DISCUSSION THREAD

**Discussion by tshoultes** [Desuuuu/DGUS-reloaded-Klipper#30] [Q&A] [answered]
Does any one know how i could rewire the default printer touch screen to go into the SKR 2? i want to use this change to make the screen work but i decided to  upgrade to the skr 2 because the SSR failed and my motherboard and SSR burned up. Would like to keep the functionality of the screen if possible, just don't know how i would rewire

**Comment by DrFate09** [comment, marked as answer]
I used this as a reference when rewiring my CR10s Pro screen to my SKR Pro 1.2, works perfectly.

https://imgur.com/a/SQtz1BI

**Reply by tshoultes** [reply]
Thanks this did help, ended up getting it working, had to wire 

Screen:                SKR 2:
5V             to       5V
GND         to       GND
TX2          to        PA10
RX2          to       PA9
